lib/App.js

```diff
@@ -1369,9 +1443,10 @@ class App {
     await fse.ensureDir( path.join(driverPath, 'assets') );
     await fse.ensureDir( path.join(driverPath, 'assets', 'images') );
 
+    const fileExtension = this.usesTypeScript() ? '.ts' : '.js';
     let templatePath = path.join(__dirname, '..', 'assets', 'templates', 'app', 'drivers');
-    await copyFileAsync( path.join(templatePath, 'driver.js'), path.join(driverPath, 'driver.js') );
-    await copyFileAsync( path.join(templatePath, 'device.js'), path.join(driverPath, 'device.js') );
+    await copyFileAsync( path.join(templatePath, `driver${fileExtension}`), path.join(driverPath, `driver${fileExtension}`) );
+    await copyFileAsync( path.join(templatePath, `device${fileExtension}`), path.join(driverPath, `device${fileExtension}`) );
```

Comment:
Ugh, I'd like to ignore the file extension and just copy the files over from another directory (a `templates/typescript` or `templates/javascript` . Not sure if thats possible somehow :/

Reply:
EDIT: I'm pretty sure what I'm suggesting is much trickier for little to no gain. ignore for now